src/site/content/en/fast/lazy-loading-images/index.md

```diff
@@ -258,12 +207,21 @@ document.addEventListener("DOMContentLoaded", function() {
   previewSize: 0
 } %}
 
-As indicated earlier, if you need Internet Explorer support for lazy-loading of background images,
-you will need to polyfill the Intersection Observer code, due to lack of support in that browser.
+### Effects on Largest Contentful Paint (LCP)
+
+Lazy loading is a great optimization that reduces both overall data usage and network contention during startup by deferring the loading of images to when they're actually needed. This can improve startup time, and reduce processing on the main thread by reducing time needed for image decodes.
+
+However, lazy loading is a technique that can affect your website's [Largest Contentful Paint LCP](/lcp/) in a negative way if you're too eager with the technique. One thing you will want to avoid is lazy loading images that are in the viewport during startup.
+
+When using JavaScript-based lazy loaders, you will want to avoid lazy loading in-viewport images because these solutions often use a `data-src` or `data-srcset` attribute as a placeholder for `src` and `srcset` attributes. The problem here is that the loading of these images will be delayed because [the browser preload scanner can't find them during startup](/preload-scanner/#lazy-loading-with-javascript).
+
+Even using browser-level lazy loading to lazy load an in-viewport image can backfire. When `loading="lazy"` is applied to an in-viewport image, [that image's priority will be lowered](/optimize-lcp/#optimize-the-priority-the-resource-is-given), which can affect a page's LCP.
```

Comment:
Literally just updated the linked text to change it saying the priority will be lowered (it won't - it will be delayed).

```suggestion
Even using browser-level lazy loading to lazy load an in-viewport image can backfire. When `loading="lazy"` is applied to an in-viewport image, [that image will be delayed until the browser knows for sure it's in the viewport](/optimize-lcp/#optimize-the-priority-the-resource-is-given), which can affect a page's LCP.
```